lib/debezium/schema_test.go

```diff
@@ -85,11 +85,6 @@ func TestField_ToKindDetails(t *testing.T) {
 		assert.Equal(t, typing.Integer, Field{Type: Int32}.ToKindDetails())
 		assert.Equal(t, typing.Integer, Field{Type: Int64}.ToKindDetails())
 	}
-	{
```

Comment:
Is this change intentional?

Reply:
Deleting this? No, must've happened from the merge. Thanks for the catch